cmd/rebasehelpers/commitchecker/validate.go

```diff
@@ -28,11 +28,11 @@ UPSTREAM commits are validated against the following regular expression:
 
 Examples of valid summaries:
 
-  UPSTREAM: 12345: An origin change
-  UPSTREAM: docker/docker: 12345: A docker change
+  UPSTREAM: 12345: A kube fix
+  UPSTREAM: github.com/coreos/etcd: 12345: An etcd fix
```

Comment:
I'm not yet seeing how we can without maintaining list of every base account name we depend on (e.g. coreos).